doc/build/core/event.rst

```diff
@@ -140,6 +140,17 @@ this value can be supported::
     # it to use the return value
     listen(UserContact.phone, "set", validate_phone, retval=True)
 
+Multiprocessing Support
+-----------------------
+
+The event system supports multiprocessing in a limited manner.
+The parent process can listen to events from within itself or child processes,
+assuming the child process was spawned after the parent has begun listening.
```

Comment:
that's not accurate either.   if an event happens in the child process, it is only intercepted in that child process and nowhere else.

Reply:
Okay thank you, I have removed this info